### PR TITLE
Remove more graphics

### PR DESCRIPTION
**Pull request by cpeel**
Removes a few more graphics from the codebase -- mostly unused ones.

The single-pixel gifs have not been accessed on PROD in the past month and I can't find them used anywhere in the codebase. The originating commit for the lot isn't very helpful either:

```
commit 0efb0dfdb713e69e26d58b9910a578872290e0cf
Author: jmdyck <>
Date:   Fri Aug 29 06:02:12 2003 +0000

    Blah
```

You can see the change in the lowered rank code by looking at the [cpeel profile](https://www.pgdp.org/~cpeel/c.branch/remove-more-graphics/stats/members/mdetail.php?id=29&tally_name=P1) in the [remove-more-graphics](https://www.pgdp.org/~cpeel/c.branch/remove-more-graphics) sandbox.

### CODE REVIEW

**Comment by chrismiceli** [comment]
> 
> 
> Removes a few more graphics from the codebase -- mostly unused ones.
> 
> The single-pixel gifs have not been accessed on PROD in the past month and I can't find them used anywhere in the codebase. The originating commit for the lot isn't very helpful either:
> 
> ```
> commit 0efb0dfdb713e69e26d58b9910a578872290e0cf
> Author: jmdyck <>
> Date:   Fri Aug 29 06:02:12 2003 +0000
> 
>     Blah
> ```
> 
> You can see the change in the lowered rank code by looking at the [cpeel profile](https://www.pgdp.org/~cpeel/c.branch/remove-more-graphics/stats/members/mdetail.php?id=29&tally_name=P1) in the [remove-more-graphics](https://www.pgdp.org/~cpeel/c.branch/remove-more-graphics) sandbox.

I was always wondering about these!

**Review by chrismiceli — APPROVED** [review]

File: stats/includes/common.inc [hunk 1]
```diff
@@ -29,21 +25,24 @@ function showChangeInRank( $previous_rank, $current_rank )
...
             // (i.e., their rank has worsened; they have moved 'down' the chart).
             $color = 'red';
-            $arrow = 'down';
-            $sign  = '-';
-            $abs   = $current_rank - $previous_rank;
+            $change = $previous_rank - $current_rank;
+            $sign = ''; // negative sign is automatic
         }
-        $snippet = "&nbsp;<span style='color: $color'>(<img src='".$GLOBALS['code_url']."/graphics/$arrow.gif' border='0' alt='$sign$abs'>&nbsp;".number_format($abs).")</span>";
+        $snippet = "&nbsp;<span style='color: $color;'>($sign".number_format($change).")</span>";
```
Comment (srjfoo) [inline stats/includes/common.inc:45]: Do you want me to pull this out into `layout.less` in a later commit?

**Review by srjfoo — APPROVED** [review]
I remembered square blocks being used in the wiki to indicate progres in localization, but I just checked and those all seem to be included as files in the wiki, so removing them here shouldn't be a problem.

File: stats/includes/common.inc (hunk 1 above)
Comment (cpeel) [reply stats/includes/common.inc:45]: Yes, assuming we care :D

Comment (srjfoo) [reply stats/includes/common.inc:45]: :D

**Comment by jmdyck** [comment]
The single-pixel gifs were used in `stats/renewals_status.php`, which was removed 2012-12-26. See, e.g. `git log -S 66CC66`. (`git log -S` and `git log -G` are ridiculously useful.)

As for the originating commit: yup, not very helpful. Mea maxima culpa.

**Pull request merged**